resources/schema.json

```diff
@@ -56,33 +56,88 @@
       "description": "extra packages to install. Not all of these may exist depending on the resource type and if they are needed",
       "properties": {
         "conda": {
-          "type": "string",
-          "description": "Packages to install from conda"
+          "type": "object",
+          "description": "Packages to install from conda",
+          "properties": {
+            "packages": {
```

Comment:
thoughts on having this be called `install`?

Reply:
I do not have a preference on that! Maybe install is more general which is better

Reply:
Ok I prefer `install` so it reads like:

```
conda.install = "-c conda-forge pandas numpy"
conda.use_mamba = true
```